lib/recurly.js

```diff
@@ -35,6 +36,10 @@ const defaults = {
   publicKey: '',
   parent: true,
   cors: false,
+  fraud: {
+    dataCollector: false,
```

Comment:
It seems a bit overly nested since `kount` and `litle` are very likely to never have another option. It also forces usage of these config options to check for another level of presence, or at least the base Recurly class needs to always make sure the structure is there. I'm fine with it though if it makes it easier to understand.